sentences/nl/homeassistant_HassTurnOff.yaml

```diff
@@ -4,6 +4,7 @@ intents:
     data:
       - sentences:
           - <doe> <name> [<naar>] uit
```

Comment:
you can make `<doe>` optional instead of adding another sentence.
So remove the added sentences and change `<doe>` in the existing sentence to `[<doe>]`